packages/lesswrong/server/resolvers/userResolvers.ts

```diff
@@ -301,3 +301,138 @@ createPaginatedResolver({
     return await context.repos.users.getSubscriptionFeedSuggestedUsers(currentUser._id, limit);
   }
 });
+
+addGraphQLSchema(`
+  type NetKarmaChangesForAuthorsOverPeriod {
```

Comment:
I believe this can be inlined into the `schema` field in the `defineQuery` props